### PR TITLE
Add function drop all bots

### PR DESCRIPTION
**Pull request by diamante0018**

### CODE REVIEW

File: src/game/game.cpp [hunk 1]
```diff
@@ -2,5 +2,13 @@
 
 namespace game
 {
+	void SV_GameDropClient(int clientNum, const char* reason)
+	{
+		assert(sv_maxclients->current.integer >= 1 && sv_maxclients->current.integer <= 18);
```
Comment (fedddddd) [inline src/game/game.cpp:7]: Why this?

**Review by fedddddd — CHANGES_REQUESTED** [review]

File: src/game/game.cpp (hunk 1 above)
Comment (diamante0018) [reply src/game/game.cpp:7]: It's a check to make sure that sv_maxclients is not out of bounds. I just replicated the game's original function.
If you don't want it I can remove it

Comment (fedddddd) [reply src/game/game.cpp:7]: Don't really see the point of it since the dvar already has limits so i guess remove it

Comment (diamante0018) [reply src/game/game.cpp]: Removed

**Pull request merged**